### ISSUE TITLE
Additional Columns in Service Alert Metrics

### ISSUE THREAD

**Issue by terrywood02** [target/goalert#3608]
**What problem would you like to solve? Please describe:**
Reporting on who is submitting and handling alerts over an extended time is very manual. 

**Describe the solution you'd like:**
Add additional columns to the Service Alert Metrics to include the submitter, acknowledger and closer. 

**Describe alternatives you've considered:**
Tried manually gathering data but a lot to filter through. Also tried GraphQL reporting but no success. 

**Additional context:**

**Comment by mastercactapus** [comment]
To clarify, this would just be for the table, correct?

Something like this?
| Created By | Acked By | Closed By |
| --- | --- | --- |
| Bob | Joe, John | Int. Key (test) |
| Int. Key (grafana) |  | Int. Key (grafana) |
| Bob | Joe | John |

**Comment by terrywood02** [comment]
Exactly! -TerryOn Jan 17, 2024, at 11:16 AM, Nathaniel Caza ***@***.***> wrote:﻿
To clarify, this would just be for the table, correct?
Something like this?



Created By
Acked By
Closed By




Bob
Joe, John
Int. Key (test)


Int. Key (grafana)

Int. Key (grafana)


Bob
Joe
John
